gpbp/distance.py

```diff
@@ -294,7 +294,7 @@ def population_served(
                 .apply(list)
                 .to_dict()
             )
-    serve_df = pd.DataFrame(index=fac_gdf["ID"].values, data=serve_dict).applymap(
+    serve_df = pd.DataFrame(index=fac_gdf["ID"].values, data=serve_dict).map(
```

Comment:
This is not related to the update, but it is a warning that we were getting for a while from `pandas`. `applymap` has been deprecated.